AUTHORS

```diff
@@ -1,2 +1,2 @@
--Goncalo Pinheira
```

Comment:
No, I did not want to, actually. (I'd even prefer it as MarkDown) Apparently this happened in my repository cleanup